src/js/components/DeploymentComponent.jsx

```diff
@@ -16,10 +19,75 @@ var DeploymentComponent = React.createClass({
 
   getInitialState: function () {
     return {
-      loading: false
+      loading: false,
+      continueButtonsLoadingState: {}
     };
   },
 
+  componentWillMount: function () {
+    DeploymentStore.on(DeploymentEvents.CONTINUE_MIGRATION_SUCCESS,
+      this.onContinueMigrationSuccess);
+    DeploymentStore.on(DeploymentEvents.CONTINUE_MIGRATION_ERROR,
+      this.onContinueMigrationError);
+  },
+
+  componentWillUnmount: function () {
+    DeploymentStore.removeListener(DeploymentEvents.CONTINUE_MIGRATION_SUCCESS,
+      this.onContinueMigrationSuccess);
+    DeploymentStore.removeListener(DeploymentEvents.CONTINUE_MIGRATION_ERROR,
+      this.onContinueMigrationError);
+  },
+
+  getContinueButton: function (action) {
+    if (action.isWaitingForUserDecision === true) {
+      let continueButtonClasses = classNames("btn btn-xs btn-success", {
+        disabled: !!this.state.continueButtonsLoadingState[action.app]
+      });
+      return (
+        <button onClick={this.handleContinueMigration.bind(this, action.app)}
+          className={continueButtonClasses}>
+          Continue
+        </button>
+      );
+    }
+
+    return null;
+  },
+
+  handleContinueMigration: function (appId, event) {
+    event.preventDefault();
+    var app = AppsStore.getCurrentApp(appId);
+    var labels = app.labels;
+    var service = null;
```

Comment:
This variable would profit by a more descriptive naming.
How about `dcosServiceName` ?